### PR TITLE
Add --use-hermes arg to bypass hermes checks and force enable it

### PR DESCRIPTION
**Pull request by youedd**
With this PR users can bypass hermes checks and force enable it using `--use-hermes` arg

Fixes https://github.com/microsoft/appcenter-cli/issues/2112

### CODE REVIEW

**Comment by youedd** [comment]
@microsoft-github-policy-service agree

**Review by aleksandr-dorofeev — DISMISSED** [review]

File: src/commands/codepush/release-react.ts [hunk 1]
```diff
@@ -123,6 +123,10 @@ export default class CodePushReleaseReactCommand extends CodePushReleaseCommandB
   @hasArg
   public extraHermesFlags: string | string[];
 
+  @help("Enable hermes and bypass automatic checks")
+  @longName("use-hermes")
+  public useHermes: string | string[];
```
Comment (DmitriyKirakosyan) [inline src/commands/codepush/release-react.ts:128]: Why it is not a `boolean`?

Comment (youedd) [reply src/commands/codepush/release-react.ts]: Hi @DmitriyKirakosyan, thanks for the review!
You're right it should be a `boolean`

**Review by dorofeeevs — APPROVED** [review]

**Pull request merged**